src/build/template.html

```diff
@@ -2,7 +2,7 @@
 <html lang="{process.env.LOCALE}" dir="{process.env.LOCALE_DIRECTION}">
 <head>
   <meta charset='utf-8' >
-  <meta name="viewport" content="width=device-width, viewport-fit=cover">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
My understanding is that `initial-scale=1` doesn't do anything anymore. But it doesn't hurt to be explicit I guess.

Reply:
I removed `initial-scale=1`, and it worked great! Thanks for reviewing :) `viewport-fit=cover` seems to be problematic on ios.